File: java/08_pgm_invoice_engine/src/co/edu/campusucc/poo/app/MainInvoiceEngine.java
package co.edu.campusucc.poo.app;

import co.edu.campusucc.poo.domain.Client;
import co.edu.campusucc.poo.domain.Invoice;
import co.edu.campusucc.poo.domain.Products;

public class MainInvoiceEngine {
    public static void main(String[] args) {
        
        Client clie = new Client();
        Products prod = new Products();
        Invoice invo = new Invoice();

        clie.nombre();
        clie.apellido();
        clie.id();
        prod.nombreProducto();
        prod.id();
        prod.precio();
        invo.tipo();
        invo.id();
        clie.tipoTarjeta();
        clie.numTarjeta();
        invo.total();
    }
}

File: python/java_to_python/08-pgmInvoiceEngine/MainInvoiceEngine.py
from Client import Client
from Invoice import Invoice
from Products import Products
def main():
    clie = Client()
    prod = Products()
    invo = Invoice()

    clie.nombre()
    clie.apellido()
    clie.id()
    prod.nombreProducto()
    prod.id()
    prod.precio()
    invo.tipo()
    invo.id()
    clie.tipoTarjeta()
    clie.numTarjeta()
    invo.total()

if __name__ == "__main__":
    main()